flambe/field/text.py

```diff
@@ -169,13 +171,16 @@ def setup(self, *data: np.ndarray) -> None:
                         if token in model:
                             self.vocab[token] = index = index + 1
                             embeddings_matrix.append(torch.tensor(model[token]))
-                        elif self.unk_init_all:
-                            # Give every OOV it's own embedding
-                            self.vocab[token] = index = index + 1
-                            embeddings_matrix.append(torch.randn(model.vector_size))
                         else:
-                            # Collapse all OOV's to the same token id
-                            self.vocab[token] = self.vocab[self.unk]
+                            if self.unk_init_all:
```

Comment:
What is going on here? Why did you indent this in? Now you may assign two ID's to the same token